docker/glibc/build_scripts/CentOS-source.repo

```diff
@@ -0,0 +1,7 @@
+[base-source]
+name=CentOS-6.10 - Base SRPMS
+baseurl=http://vault.centos.org/6.10/os/Source/
+gpgcheck=1
+gpgkey=http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-6
```

Comment:
URL for gpgkey is wrong (and http).
See here: https://www.centos.org/keys/#centos-6-signing-key

links to here: https://www.centos.org/keys/RPM-GPG-KEY-CentOS-6

Reply:
Now using local gpgkey